app/Http/Controllers/LikedPicturesController.php

```diff
@@ -0,0 +1,104 @@
+<?php
+
+namespace Proto\Http\Controllers;
+
+use Auth;
+use Illuminate\Database\Eloquent\Builder;
+use Illuminate\Http\JsonResponse;
+use Illuminate\View\View;
+use Proto\Models\Photo;
+
+class LikedPicturesController extends Controller
+{
+    /**
+     * @return View
+     */
+    public function show()
+    {
+        $likedPhotos = Photo::whereHas('likes', function (Builder $query) {
+            $query->where('user_id', Auth::user()->id);
+        })->orderBy('date_taken', 'asc')->orderBy('id', 'asc')->paginate(24);
+
+        $album = new \stdClass();
+        $album->title = 'My liked photos';
+        $album->name = 'My liked photos';
+        $album->date_taken = \Carbon::today()->timestamp;
+        $album->event = null;
+
+        if ($likedPhotos->count()) {
```

Comment:
It is nicer to use a guard statement before creating the album object. Something like:
```PHP
if ($likedPhotos->empty()) { 
    abort(404) 
}
```